server/chat-plugins/info.js

```diff
@@ -2292,7 +2292,7 @@ const commands = {
 		if (!target) return this.parse('/help htmlbox');
 		target = this.canHTML(target);
 		if (!target) return;
-
+		target = target.replace(/\n/g, '');
```

Comment:
This should at least be `' '` – presence/absence of whitespace is significant in HTML.

This also affects `<pre>` blocks, which is harder to deal with.

Reply:
It looks like `&#10;` is the safest replacement.

Reply:
Although you have to do something smarter than a straight replacement – replace with a space inside tags and `&#10;` outside of tags.

Reply:
Is there a need for all that? We might as well "escape" ``\n`` as ``\f`` and have the client convert it back.

Reply:
The HTML entity removes the need to unescape.